src/main/java/frc/robot/Robot.java

```diff
@@ -28,6 +30,7 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+    m_robotContainer.limelight.setLEDs(LedValues.ON);
```

Comment:
remove all of the limelight stuff from Robot.java

Reply:
didn't we decide this was the way to fix it? or was it a sketchy way